docs/install/install.rst

```diff
@@ -1,51 +1,68 @@
 .. meta::
-  :description: TransferBench documentation 
-  :keywords: TransferBench, API, ROCm, HIP
+  :description: TransferBench is a utility to benchmark simultaneous transfers between user-specified devices (CPUs or GPUs)
+  :keywords: Build TransferBench, Install TransferBench, API, ROCm, HIP
+
+.. _install-transferbench:
 
 ---------------------------
-TransferBench installation
+Installing TransferBench
 ---------------------------
 
-The following software is required to install TransferBench:
+This document provides information required to install and build TransferBench.
 
-* ROCm stack installed on the system (HIP runtime)
-* `libnuma` installed on the system
+Prerequisite
+---------------
 
---------------------------
-Building TransferBench
---------------------------
+* Install ROCm stack on the system to obtain HIP runtime
```

Comment:
HIP should be hyperlinked to the HIP docs. Perhaps a very short description would be good here, too

Reply:
Added hyperlink